File: src/CodefictionApi/Server/Data/Contracts/IContent.cs
using System;

namespace Codefiction.CodefictionTech.CodefictionApi.Server.Data.Contracts
{
    public interface IContent
    {
        int Id { get; set; }
        string Title { get; set; }
        string Slug { get; set; }
        string YoutubeUrl { get; set; }
        string ShortDescription { get; set; }
        string LongDescription { get; set; }
        string[] Attendees { get; set; }
        string[] Tags { get; set; }
        Relation[] Relations { get; set; }
        DateTime PublishDate { get; set; }
    }
}

File: src/CodefictionApi/CodefictionApp/app/models/contracts/IContent.ts
import { IRelation } from './IRelation';

export interface IContent {
  id: number;
  title: string;
  slug: string;
  youtubeUrl: string;
  shortDescription: string;
  longDescription: string;
  attendees: string[];
  tags: string[];
  relation: IRelation[];
  publishDate: Date;
}
